Microsoft.Toolkit.Uwp.UI.Controls/GridSplitter/GridSplitter.Events.cs

```diff
@@ -254,6 +254,12 @@ private bool HorizontalMove(double horizontalChange)
             // if sibling column has fixed width then resize it
             else if (!IsStarColumn(SiblingColumn))
             {
+                // Would adding to this column make the current column violate the MinWidth?
```

Comment:
I'd assume we'd also need similar logic for the `VerticalMove` as well?

Reply:
ping @skendrot